templates/.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,12 @@
+name: Auto approve
+
+on: pull_request
```

Comment:
It seems to me this needs to trigger on some kind fo status check event since we only want to approve after tests pass, no?

Reply:
Aha, like what you have below:
```
on:
  check_suite:
    types:
      - completed
```